Patterns/Swedish/Swedish-TimeZone.yaml

```diff
@@ -348,6 +646,102 @@ AmbiguousTimezoneList: !list
 AbbrToMinMapping: !dictionary
  types: [ string , int ]
  entries:
+   # Swedish extensions
+   acwst: 525
+   almt: 360
+   #amt: 240
+   #amt: -240
+   anat: 720
```

Comment:
It looks like we're still missing the ambiguous resolution for some cases. Here in your file there are two commented out AMT, which suggests AMT is an ambiguous abbreviation, so there should be an entry for amt: -1000 to represent that.
AST and PYT are other cases. Are there more?
If these are not relevant in the target language, and you don't want/need to also accept English abbreviations, then the commented out duplicates could be deleted.